assets/js/base/components/quantity-selector/style.scss

```diff
@@ -28,27 +27,27 @@
 	// Extra label for specificity needed in the editor.
 	input.wc-block-components-quantity-selector__input {
 		@include font-size(regular);
-		order: 2;
-		min-width: 40px;
-		flex: 1 1 auto;
-		border: 0;
-		padding: 0.4em 0;
-		margin: 0;
-		text-align: center;
+		-moz-appearance: textfield;
```

Comment:
I know that it is not your change, but I guess that we could use [`appearance`](https://caniuse.com/css-appearance). What do you think?